server/app/assets/javascripts/admin_programs.ts

```diff
@@ -0,0 +1,58 @@
+class AdminPrograms {
+  private static PROGRAM_ADIN_LIST_SELECTOR = '.cf-admin-program-card-list'
+  private static PROGRAM_CARDS_SELECTOR = '.cf-admin-program-card'
+  private static NAME_ATTRIBUTE = 'data-name'
+  private static LAST_UPDATED_MILLIS = 'data-last-updated-millis'
+
+  constructor() {
+    const cardsParent = document.querySelector(
+      AdminPrograms.PROGRAM_ADIN_LIST_SELECTOR
+    )
+    if (!cardsParent) {
+      return
+    }
+    try {
+      this.sortCards(cardsParent as HTMLElement)
+    } finally {
+      // Make sure to always show the cards, even
+      // if there was an error while sorting.
+      cardsParent.classList.remove('invisible')
```

Comment:
nit: If this is invisible at first, it would be good to have a placeholder that says "loading" or something.  That way if the user doesn't have js enabled, it won't just look like there are 0 programs

Reply:
Added a placeholder since it doesn't hurt anything to have. RE JS enablement, I spoke with @bion about this a bit back, who indicated that we can assume that JS is required for CiviForm.

Reply:
especially for the admin view we can make convenient assumptions: JS is enabled, they're on desktop not mobile, decent internet connection, etc